### ISSUE TITLE
Sitemap generation uses full store_code instead of stripped store_code

### ISSUE THREAD

**Issue by TimQSO** [experius/Magento2-Module-Multiple-Website-Store-Code-Url#6]
Magento 2.3.3
Nginx 1.17
PHP 7.3

When enabling the module it succesfully strips the website_code from the store_code, but the sitemap.xml that is generated contains the full store_code including the website_code. For example:

```
<url>
<loc>https://www.mywebsite.com/mysite_nl/aanbiedingen</loc>
<lastmod>2020-02-04T01:15:11+00:00</lastmod>
<changefreq>daily</changefreq>
<priority>0.5</priority>
</url>
```

But this should be:

```
<url>
<loc>https://www.mywebsite.com/nl/aanbiedingen</loc>
<lastmod>2020-02-04T01:15:11+00:00</lastmod>
<changefreq>daily</changefreq>
<priority>0.5</priority>
</url>
```

I can use this as a cron job as a workaround in the mean time, but it should be taken care of:

` sed -i 's/mysite_nl/nl/g' sitemap.xml`

**Comment by lewisvoncken** [comment]
@GuiltyNL this will be solved in the upcoming release

**Comment by lewisvoncken** [comment]
@GuiltyNL this is solved in version 2.1.0
